Kconfig.xtos-dbg

```diff
@@ -12,3 +12,12 @@ config DEBUG_BLOCK_FREE
 	  already freed block of memory. Enabling this feature increases
 	  number of memory writes and reads, due to checks for memory patterns
 	  that may be performed on allocation and deallocation.
+
+config DEBUG_FORCE_COHERENT_BUFFER
+	bool "Force the allocator to allocate coherent buffer only"
```

Comment:
means uncached buffer ?  I know cache coherence but what is coherence buffer ?

Reply:
Yes, so coherent (a generic term) means uncache (coherent) memory on Intel HW.